doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/launching-clusters/on-premises.md

```diff
@@ -0,0 +1,116 @@
+# Launching On-Premises Cluster
+
+This docs describes how to set up an on-premises Ray cluster, i.e., to run Ray on bare metal machines, or in a private cloud. We provide two ways to start an on-premises cluster.
+
+* You can **manually set up** the Ray cluster by installing the Ray package and starting the Ray processes on each node. 
+* Alternatively, if you know all the nodes in advance and have ssh access to them, you should start the ray cluster using **cluster-launcher**.
+
+
+## Manually Set up a Ray Cluster 
+This section assumes that you have a list of machines and that the nodes in the cluster can communicate with each other. It also assumes that Ray is installed on each machine.  You can use pip to install it, or follow [install ray](https://docs.ray.io/en/latest/ray-overview/installation.html) for more detailed instructions.
+
+```
+# install ray cluster launcher, which is part of ray command line tool.
```

Comment:
article
`the ray command line tool.`

Reply:
Actually, we don't need the cluster launcher for manual set up.
I think we just want
```
# install ray
pip install -U ray
```